examples/triton-docker-combined/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "account" {}
+variable "triton_url" {}
+variable "key_path" {}
+variable "key_id" {}
+variable "cert_path" {}
```

Comment:
Should this name more clearly differentiate itself from `key_path`? Perhaps both names should be changed?